config/v1/types_image.go

```diff
@@ -67,6 +84,19 @@ type ImageSpec struct {
 	// internal cluster registry.
 	// +optional
 	RegistrySources RegistrySources `json:"registrySources"`
+
+	// imageStreamImportMode controls the import mode behaviour of imagestreams.
+	// It can be set to `Legacy` or `PreserveOriginal`. If this value is specified,
+	// this setting is applied to all newly created imagestreams which do not have the
+	// value set. ImportModeLegacy indicates that the legacy behaviour should be used.
+	// For manifest lists, the legacy behaviour will discard the manifest list and import a single
+	// sub-manifest. In this case, the platform is chosen in the following order of priority:
+	// 1. tag annotations; 2. control plane arch/os; 3. linux/amd64; 4. the first manifest in the list.
+	// This mode is the default. ImportModePreserveOriginal indicates that the original manifest
+	// will be preserved. For manifest lists, the manifest list and all its sub-manifests will be imported.
```

Comment:
```suggestion
	// imageStreamImportMode controls the import mode behaviour of imagestreams.
	// It can be set to `Legacy` or `PreserveOriginal` or the empty string. If this value is specified,
	// this setting is applied to all newly created ImageStreams which do not have the
	// value set. `Legacy` indicates that the legacy behaviour should be used.
	// For manifest lists, the legacy behaviour will discard the manifest list and import a single
	// sub-manifest. In this case, the platform is chosen in the following order of priority:
	// 1. tag annotations; 2. control plane arch/os; 3. linux/amd64; 4. the first manifest in the list.
	// `PreserveOriginal` indicates that the original manifest
	// will be preserved. For manifest lists, the manifest list and all its sub-manifests will be imported.
	// When empty, the `Legacy` behaviour is applied.
```
Just want to make this flow a bit with the language users will see. They care about the yaml naming, not the go naming

Reply:
Right ok, can you update my suggestion then with how you would word that? Probably something along the lines of
```
// When empty, the mode is determined based on the payload image specified in the ClusterVersion resource. 
```

Reply:
made the appropriate changes.